core/templates/dev/head/components/summary_tile/ExplorationSummaryTileMobileDirective.js

```diff
@@ -0,0 +1,141 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for an exploration summary tile.
+ */
+
+oppia.directive('explorationSummaryTileMobile', [function() {
```

Comment:
This and the following file seem like a big copy paste, which is generally a no-no, since if a bug is discovered and gets fixed in one directive, it's unlikely to get fixed in the other. Is it not possible to modify the existing summary tile directive by thinking more specifically about what parts need to be changed and passing in args as needed so that the branching only happens in limited parts of the code?

Or are the two directives too different to do this? It would be helpful to see your reasons why -- what are the main points of difference?

Happy to chat about this if you like, too -- feel free to ping in the main gitter channel.